packages/hooks/src/index.js

```diff
@@ -7,34 +6,30 @@ import createCurrentHook from './createCurrentHook';
 import createDoingHook from './createDoingHook';
 import createDidHook from './createDidHook';
 
-// Add action/filter functions.
-export const addAction = createAddHook( HOOKS.actions );
-export const addFilter = createAddHook( HOOKS.filters );
 
-// Remove action/filter functions.
-export const removeAction = createRemoveHook( HOOKS.actions );
-export const removeFilter = createRemoveHook( HOOKS.filters );
-
-// Has action/filter functions.
-export const hasAction = createHasHook( HOOKS.actions );
-export const hasFilter = createHasHook( HOOKS.filters );
-
-// Remove all actions/filters functions.
-export const removeAllActions = createRemoveHook( HOOKS.actions, true );
-export const removeAllFilters = createRemoveHook( HOOKS.filters, true );
-
-// Do action/apply filters functions.
-export const doAction     = createRunHook( HOOKS.actions );
-export const applyFilters = createRunHook( HOOKS.filters, true );
-
-// Current action/filter functions.
-export const currentAction = createCurrentHook( HOOKS.actions );
-export const currentFilter = createCurrentHook( HOOKS.filters );
-
-// Doing action/filter: true while a hook is being run.
-export const doingAction = createDoingHook( HOOKS.actions );
-export const doingFilter = createDoingHook( HOOKS.filters );
-
-// Did action/filter functions.
-export const didAction = createDidHook( HOOKS.actions );
-export const didFilter = createDidHook( HOOKS.filters );
+export const Hooks = () => {
+	const HOOKS = {
+		actions: {},
+		filters: {},
+	};
+
+	return {
+		'addAction':        createAddHook( HOOKS.actions ),
+		'addFilter':        createAddHook( HOOKS.filters ),
+		'removeAction':     createRemoveHook( HOOKS.actions ),
+		'removeFilter':     createRemoveHook( HOOKS.filters ),
+		'hasAction':        createHasHook( HOOKS.actions ),
+		'hasFilter':        createHasHook( HOOKS.filters ),
+		'removeAllActions': createRemoveHook( HOOKS.actions, true ),
+		'removeAllFilters': createRemoveHook( HOOKS.filters, true ),
+		'doAction':         createRunHook( HOOKS.actions ),
+		'applyFilters':     createRunHook( HOOKS.filters, true ),
+		'currentAction':    createCurrentHook( HOOKS.actions ),
+		'currentFilter':    createCurrentHook( HOOKS.filters ),
+		'doingAction':      createDoingHook( HOOKS.actions ),
+		'doingFilter':      createDoingHook( HOOKS.filters ),
+		'didAction':        createDidHook( HOOKS.actions ),
+		'didFilter':        createDidHook( HOOKS.filters ),
+		'HOOKS':            HOOKS,
```

Comment:
minor: we could remove the quotes from the keys here.

Reply:
ah, right. will fix.

Reply:
addressed in https://github.com/WordPress/packages/pull/40/commits/031fdfddb6484329c6d884edd8440de69a256e58